src/inputs/phoneInput/phoneInputCountryDropdown.jsx

```diff
@@ -55,7 +55,7 @@ class PhoneInputCountryDropdown extends React.PureComponent {
 
         if (prevProps.value !== value && prevState.selectedOption.value !== value) {
             this.setState({
-                selectedOption: _.find(options, (o) => !o.divider && o.value === value),
+                selectedOption: _.find(options, (o) => !o.divider && o.value && o.value === value),
```

Comment:
Do we need these changes here?

Reply:
this is just being extra defensive again in the case of not having a value. But you might have a point that at least in theory there shouldn't be any path that would lead us here without a value on the option item. I can remove, let me know.